src/scripts/main.js

```diff
@@ -1,3 +1,298 @@
 'use strict';
 
-// write code here
+const MIN_AGE = 18;
+const MAX_AGE = 100;
```

Comment:
One test is failing as the MAX_AGE should be 90
![image](https://github.com/mate-academy/js_employees_table_DOM/assets/78635631/9867d6cb-3807-4629-849f-59a68ff880cb)

Reply:
Where can i read test requirements for the future tasks?

Reply:
usually it's mention in task instruction 
![image](https://github.com/mate-academy/js_employees_table_DOM/assets/126968958/782c90dd-4788-4869-83ab-41af8626b815)
you can also check tests in cypress/integration